blogfrontend/src/components/Login.js

```diff
@@ -0,0 +1,59 @@
+import React,{useState} from 'react'
+import { Link } from "react-router-dom";
+
+const Login = () => {
+    const [username,setUserName]=useState({
+       username:"",
+       password:"" 
+    })
+    const handlChanges=(e)=>{
+        const {name,value}=e.target
+        console.log(name,value)
+        setUserName({
+            ...username,
+            [name]:value
+        })
+        
+    }
+    const handleSubmit=(e)=>{
+        e.preventDefault()
+        console.log("datZ",e.target.name);
+        if(!username.username){
+          alert('Enter your  username')
+        }else if(!username.password){
```

Comment:
if you are using if and else if please follow the indentation which is shown below:

`If (condition1) {
    // code
} else if (condition2) {
   // Code
}
`